src/vs/workbench/contrib/search/browser/searchModel.ts

```diff
@@ -1168,11 +1214,61 @@ export class SearchResult extends Disposable {
 
 		return retEvent;
 	}
+
+	private onDidVisibleEditorsChange(): void {
+		const visibleNotebookEditors = this.editorService.visibleEditorPanes
```

Comment:
Notebook editor widgets can be invisible and alive, I wonder if visible notebook editors are too limited.